<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>物流业务和用户管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/res/3.jpg);</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/res/1.JPG);
background-image: url(:/new/res/3.jpg);</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="loginpage">
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>501</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-image: url(:/new/res/2.JPG);</string>
      </property>
      <widget class="QLabel" name="name">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>100</y>
         <width>48</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>用户</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_name">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>100</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="password">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>170</y>
         <width>48</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_pw">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>10</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QPushButton" name="LoginButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>260</y>
         <width>411</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QPushButton" name="RegisterButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>320</y>
         <width>411</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pagemgm">
     <widget class="QLineEdit" name="d_info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请输入想要签收的快递单号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SingforD">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>签收快递</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>541</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="d_uinfo">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请输入想要查找的快递信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dsearch">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>查找快递</string>
      </property>
     </widget>
     <widget class="QPushButton" name="send">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>180</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>发送快递</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="searchdelivery">
     <widget class="QLineEdit" name="le_number">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入想要查询的快递相关信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchbutton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="searchdtable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>641</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>121</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>欢迎！</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>80</y>
        <width>361</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/new/res/740.JPG);
background-repeat:no-repeat;
background-position:center;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="searchuser">
     <widget class="QLineEdit" name="le_user">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>请输入想要查询的用户相关信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="searchbutton_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="searchutable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>671</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionlogin"/>
    <addaction name="actionchange_password"/>
    <addaction name="actiondeliverymanage"/>
    <addaction name="actionbalance"/>
    <addaction name="actionlogout"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>管理员</string>
    </property>
    <addaction name="actionsearch_delivery"/>
    <addaction name="actionsearch_user"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlogin">
   <property name="text">
    <string>登录</string>
   </property>
   <property name="iconText">
    <string>登录</string>
   </property>
   <property name="toolTip">
    <string>登录</string>
   </property>
  </action>
  <action name="actionchange_password">
   <property name="text">
    <string>修改密码</string>
   </property>
   <property name="iconText">
    <string>修改密码</string>
   </property>
   <property name="toolTip">
    <string>修改密码</string>
   </property>
  </action>
  <action name="actionbalance">
   <property name="text">
    <string>余额管理</string>
   </property>
   <property name="iconText">
    <string>余额管理</string>
   </property>
   <property name="toolTip">
    <string>余额管理</string>
   </property>
  </action>
  <action name="actionlogout">
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="actionsearch_delivery">
   <property name="text">
    <string>查询快递</string>
   </property>
   <property name="toolTip">
    <string>查询管理</string>
   </property>
  </action>
  <action name="actiondeliverymanage">
   <property name="text">
    <string>快递管理</string>
   </property>
  </action>
  <action name="actionsearch_user">
   <property name="text">
    <string>查询用户</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
